tests/integration-service/README.md

```diff
@@ -2,25 +2,57 @@
 
 The Integration Service manages the lifecycle, deployment and integration testing of applications and components. It handles snapshots, environments, pipelines and deployments, ensuring applications run efficiently and are deployed correctly.
 
-
 ## Happy Path Tests
 
 Happy path testing describes tests that focus on the most common scenarios while assuming there are no exceptions or errors.
+
 ### Basic E2E Tests (within `integration.go`):
 
+These tests cover the end-to-end integration service workflow, verifying the successful execution of key processes such as component creation, pipeline runs, snapshot generation, and release management under ideal conditions.
+
+Checkpoints: 
+
 - Testing for successful creation of applications and components.
-- Asserting the successful creation of a snapshot after a push event.
-- Checking if the BuildPipelineRun is successfully triggered, contains the finalizer, and got completed.
+- Checking if the BuildPipelineRun is successfully triggered, contains the finalizer, and was completed.
 - Asserting the signing of BuildPipelineRun.
 - Validating the successful creation of a Snapshot, and removal of finalizer from BuildPipelineRun.
 - Verifying that all the Integration PipelineRuns finished successfully.
 - Checking that the status of integration tests were reported to the Snapshot.
-- Checking that a snapshot gets successfully marked as 'passed' after all Integration pipeline runs finished.
-- Validating the Global Candidate is updated
+- Checking that a snapshot gets successfully marked as 'passed' or 'failed' after all Integration pipeline runs finished.
+- Handling of Skipped Integration Tests.
+- Validating the Global Candidate is updated.
 - Validating the successful creation of a Release.
+- Asserting the successful creation of a snapshot after a push event.
+- Management of Pull Requests and Branches.
+- Re-running Integration Tests.
+- Finalizer Removal from Integration PipelineRuns.
+
+## Negative Test Cases
```

Comment:
Under this `Negative Test Cases` section, we are also describing some happy paths, which isn't expected, wdyt?

Overall, in this README file, I'm thinking of 2 ways to list down the details:
1. Per test suites
    * {Describe integration.go}
    * {Describe status-reporting-to-pr.go}
.
.
.

2. Per Happy and Negative paths (like we have currently)
    * Happy Paths
        * List all happy paths for each test suites
    * Negative Paths
        * List all negative paths of each test suites

Wdyt about this?

Reply:
ok, I've done something along the lines of your request for your feedback. If this layout is satisfactory I can go confirm everything and there might be another push. I just wanted to get your take on how it looks before going further.

Reply:
Looks very neat now, thank you for making the changes :raised_hands: